web-ide/ide-server/Dockerfile

```diff
@@ -11,10 +11,11 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - &&\
         procps \
         nodejs \
         libxkbfile-dev \
-        libsecret-1-dev
+        libsecret-1-dev &&\
+    apt-get remove -y --allow-remove-essential apt
 
 USER sdk
-ENV VERSION=0.11.19 \
+ENV VERSION=0.12.16 \
```

Comment:
Any chance we can move these two versions together, ideally call them DAML_VERSION (to be distinct from CODESERVER_VERSION). Actually combining them to a single version would be preferable.

Reply:
I think it's some annoyance with changing users...I could not use the arg value from the top of the file!

I'll try again, maybe I was doing something stupid.

Reply:
found a work-around, apparently you have to redefine (but you don't have to set the default value again) the ARG. This is some oddity with defining ARG before FROM, it all gets wiped after the FROM